documentation/asciidoc/computers/config_txt/common.adoc

```diff
@@ -0,0 +1,39 @@
+== Common Options
+
+=== Common Display Options
+
+==== `disable_overscan`
+
+Set `disable_overscan` to `1` to disable the default values of xref:configuration.adoc#underscan[overscan] that is set by the firmware. The default value of overscan for the left, right, top, and bottom edges is `48` for HD CEA modes, `32` for SD CEA modes, and `0` for DMT modes. The default value for `disable_overscan` is `0`.
+
+==== `hdmi_enable_4kp60` (Raspberry Pi 4 Only)
+
+By default, when connected to a 4K monitor, the Raspberry Pi 4B, 400 and CM4 will select a 30hz refresh rate. Use this option to allow selection of 60Hz refresh rates. Note, this will increase power consumption and increase the temperature of the Raspberry Pi. It is not possible to output 4Kp60 on both micro HDMI ports simultaneously.
+
+=== Common Hardware Configuration Options
+
+==== `auto_detect_cameras`
+
+With this setting enabled (set to `1`), the firmware will automatically load overlays for cameras that it recognises. New Rasperry Pi OS images from Bullseye onwards come with this setting by default.
+
+==== `auto_detect_displays`
+
+With this setting enabled (set to `1`), the firmware will automatically load overlays for displays that it recognises. New RPiOS images from Bullseye onwards come with this setting by default.
+
+==== `dtoverlay`
+
+The `dtoverlay` option requests the firmware to load a named Device Tree overlay - a configuration file that can enable kernel support for built-in and externalhardware. For example, `dtoverlay=vc4-kms-v3d` loads an overlay that enables the kernel graphics driver.
+
+As a special case, if called with no value - `dtoverlay=` - it marks the end of a list of overlay parameters. If used before any other `dtoverlay` or `dtparam` setting it prevents the loading of any HAT overlay.
+
+For more details, see xref:configuration.html#part3.1[DTBs, overlays and config.txt].
+
+==== `dtparam`
+
+Device Tree configuration files for Raspberry Pis support a number of parameters for such things as enabling I2C and SPI interfaces. Many DT overlays are configurable via the use of parameters. Both types of parameters can be supplied using the `dtparam` setting. In addition, overlay parameters can be appended to the `dtoverlay` option, separated by commas, but beware the line length limit - previously 78 characters, now 98 characters.
+
+For more details, see xref:configuration.html#part3.1[DTBs, overlays and config.txt].
+
+==== `arm_boost` (Raspberry Pi 4 Only)
+
+All Pi 400s and newer revisions of the Pi 4B are equipped with a second switch-mode power supply for the SoC voltage rail, and this allows the default turbo-mode clock to be increased from 1.5GHz to 1.8GHz. This change should be safe for all such boards, but to avoid unrequested changes for existing installations this change must be accepted by setting `arm_boost=1`. New images from Bullseye onwards come with this setting by default.
```

Comment:
We're trying (valiantly!) to remove usage of "Pi", for brand-related reasons we need to write "Raspberry Pi" in full.

Reply:
Also here with the "New RPiOS images from Bullseye onwards come with this setting by default."